client/src/cli_args.rs

```diff
@@ -172,8 +172,8 @@ impl<'a, 'b> EncointerArgs<'b> for App<'a, 'b> {
 			Arg::with_name(WHITELIST_ARG)
 				.takes_value(true)
 				.required(false)
-				.multiple(true)
-				.min_values(1),
+				.value_name("WHITELIST")
```

Comment:
I just realize that people may use spaces after the comma which breaks this argument: maybe indicate in help that no spaces may be used?